pymc/distributions/multivariate.py

```diff
@@ -2669,8 +2667,8 @@ def dist(cls, sigma=1, n_zerosum_axes=None, support_shape=None, **kwargs):
         n_zerosum_axes = cls.check_zerosum_axes(n_zerosum_axes)
 
         sigma = pt.as_tensor_variable(floatX(sigma))
-        if sigma.ndim > 0:
-            raise ValueError("sigma has to be a scalar")
+        if not all(sigma.type.broadcastable[-n_zerosum_axes:]):
+            raise NotImplementedError("sigma must have length one across the zerosumaxes")
```

Comment:
I would still raise a `ValueError` here instead of a `NotImplementedError`

Reply:
I just did this because it's a promised feature in the docstrings

Reply:
I like `ValueError` more though. Changed.